From c75d5fe89ed58542a3654e0646674c0c64e82eea Mon Sep 17 00:00:00 2001
Message-ID: <c75d5fe89ed58542a3654e0646674c0c64e82eea.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:44:00 +0100
Subject: [PATCH 11/19] gnu: go-github-com-google-gofuzz: Move to (gnu packages
 golang-check).

* gnu/packages/golang-check.scm (go-github-com-google-gofuzz): Move from
here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm       | 25 -------------------------
 2 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index f904e8b02c..9fdb5b0d9e 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -41,6 +41,31 @@ (define-module (gnu packages golang-check)
 ;;;
 ;;; Code:
 
+(define-public go-github-com-google-gofuzz
+  (let ((commit "fd52762d25a41827db7ef64c43756fd4b9f7e382")
+        (revision "0"))
+    (package
+      (name "go-github-com-google-gofuzz")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/google/gofuzz")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "1yxmmr73h0lq7ryf3q9a7pcm2x5xrg4d5bxkq8n5pxwxwyq26kw8"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/google/gofuzz"))
+      (home-page "https://github.com/google/gofuzz")
+      (synopsis "Fuzz testing library for Go")
+      (description "Gofuzz is a library for populationg Go objects with random
+values for the purpose of fuzz testing.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-golangplus-testing
   (package
     (name "go-github-com-golangplus-testing")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 04751b6a79..1e78bfd5bd 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -4655,31 +4655,6 @@ (define-public go-github-com-google-cadvisor
 containers.")
       (license license:asl2.0))))
 
-(define-public go-github-com-google-gofuzz
-  (let ((commit "fd52762d25a41827db7ef64c43756fd4b9f7e382")
-        (revision "0"))
-    (package
-      (name "go-github-com-google-gofuzz")
-      (version (git-version "0.0.0" revision commit))
-      (source
-       (origin
-         (method git-fetch)
-         (uri (git-reference
-               (url "https://github.com/google/gofuzz")
-               (commit commit)))
-         (file-name (git-file-name name version))
-         (sha256
-          (base32
-           "1yxmmr73h0lq7ryf3q9a7pcm2x5xrg4d5bxkq8n5pxwxwyq26kw8"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/google/gofuzz"))
-      (home-page "https://github.com/google/gofuzz")
-      (synopsis "Fuzz testing library for Go")
-      (description "Gofuzz is a library for populationg Go objects with random
-values for the purpose of fuzz testing.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-gorilla-css
   (package
     (name "go-github-com-gorilla-css")
-- 
2.41.0

